framework/doc/content/source/meshgenerators/CircularBoundaryCorrectionGenerator.md

```diff
@@ -0,0 +1,77 @@
+
+In a 2D mesh, a "circular boundary" consists of sides that connects a series of nodes on the circle, which actually form a polygon boundary. Due to the polygonization effect, the area within a "circular boundary" in a 2D mesh is actually smaller than a real circle with the same radius. Such a discrepancy could cause issues in some simulations that involve physics that is sensitive to volume conservation (e.g., neutronics).
+
+Therefore, a corrected radius can be used to generate the "circle-like" polygon to enforce that the polygon area is the same as the original circle without polygonization. This can be achieved through the following equations.
+
+### Moving Radial Nodes
+
+The most straightforward approach for circular correction is to move the nodes on the circular boundary in their respective radial directions (see the left sub-figure of [schematic]). In that case, the azimuthal angle intervals of the boundary sides do not change. Therefore, the algorithm is relatively simple. This is also the default approach of this mesh generator.
+
+For a polygon region used to mesh a full or partial circle, each side (with index $i$) corresponds to an azimuthal angle interval, $\theta_i$ , which is defined by the side and the center of the polygon:
+
+!equation id=polygon_area
+S_{polygon}=\frac12r_{polygon}^2\Sigma_i~\sin \theta_i
+
+with
+
+!equation id=azi
+\Theta_{total}=\Sigma_i~\theta_i
+
+$\Theta_{total}$ should be $2\pi$ for a full circle and between 0 to $2\pi$ for a partial circle. For such a full or partial real circle, the area has the following form:
+
+!equation id=circle_area
+S_{circle}=\frac{\Theta_{total}}{2}r_{circle}^2=\frac12r_{circle}^2\Sigma_i~\theta_i
+
+To enforce that $S_{polygon}=S_{circle}$, the radii of the polygon and the circle have the following relation,
+
+!equation id=relation
+r_{polygon}=\sqrt{\frac{\Sigma_i~\theta_i}{\Sigma_i~\sin \theta_i}}r_{circle}=f_{corr}r_{circle}
+
+Where $f_{corr}$ is the correction factor used in this object to ensure volume preservation.
+
+!media mesh_modifiers/circular_correction.png
+      style=display: block;margin-left:auto;margin-right:auto;width:60%;
+      id=schematic
+      caption=A schematic drawing showing the two approaches to correct the polygonization effect for a partial circular boundary
+
+### Azimuthally Displacing Nodes in the Span Direction
+The radial nodes moving approach is undoubtedly the most generalized approach for a full circular boundary. However, for a partial circular boundary, moving the two end nodes in their radial directions may deform the original shape. Therefore, moving the end nodes in the span direction of the partial circular boundary (i.e., arc) provides an alternative approach (see the right subfigure of [schematic]). Moving the end nodes in the span direction inevitably changes the azimuthal angle intervals. To make this change consistent for all the boundary sides, a scaling coefficient, $c$, is applied to every $\theta_i$.
+
```

Comment:
this is controlled by move_end_nodes_in_span_direction right?
but this only refers to the end nodes. 

and it does not convey that there is then no radial displacement right?

maybe the right parameter would be a method enum, like correction_method = "radial" or "azimuthal"

Reply:
Even if the end nodes are moved in the span direction, the other nodes will also be moved in the radial direction. The only difference is the other nodes will also be moved in azimuthal direction.

Reply:
Makes sense.